src/Playwright/BrowserContext.cs

```diff
@@ -17,6 +17,7 @@ internal partial class BrowserContext : ChannelOwnerBase, IChannelOwner<BrowserC
         private readonly TaskCompletionSource<bool> _closeTcs = new();
         private readonly Dictionary<string, Delegate> _bindings = new();
         private readonly BrowserContextInitializer _initializer;
+        private readonly ITracing _tracing;
```

Comment:
```suggestion
```

Reply:
I am following the Page._mouse pattern here.